lib/ansible/plugins/connection/paramiko_ssh.py

```diff
@@ -509,12 +531,16 @@ def _connect_uncached(self) -> paramiko.SSHClient:
             if LooseVersion(paramiko.__version__) >= LooseVersion('1.15.0'):
                 ssh_connect_kwargs['banner_timeout'] = self.get_option('banner_timeout')
 
+            passphrase = None
+            if self.get_option('private_key_file_passphrase'):
+                passphrase = self.get_option('private_key_file_passphrase')
```

Comment:
The if-block is redundant. You're essentially doing this one-liner:
```suggestion
            passphrase = self.get_option('private_key_file_passphrase') or None
```
(also, two calls to `get_option()` is probably expensive)

Reply:
not really, as it should cache the option after the first access